### PR TITLE
docs: nix installation instructions

### PR DESCRIPTION
**Pull request by kbudde**
Mention nix package in readme.

#285

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -136,7 +142,7 @@ These commands (`sync`, `render`, `all`) accept two optional arguments:
 environments and applications to process. When no arguments are provided, myks
 will use the [Smart Mode](/docs/smart-mode.md) to detect what to process.
 
-> [!TIP]  
+> [!TIP]
```
Comment (Zebradil) [inline docs/README.md:145]: > [!IMPORTANT]  
> The double space in the end of this line is important to properly form the alert block. Like this one.

```suggestion
> [!TIP]  
```

File: docs/README.md [hunk 2]
```diff
@@ -101,6 +101,12 @@ brew tap mykso/tap
 brew install myks
 ```
 
+### Nix
+
+```
+nix-shell -p myks helm git
```
Comment (Zebradil) [inline docs/README.md:107]: This made me think that we probably should just specify deps in the nix package instead. But we can do that later.

**Review by Zebradil — APPROVED** [review]
Whoa! Who's back! :-)

File: docs/README.md (hunk 2 above)
Comment (kbudde) [reply docs/README.md:107]: I wouldn't do it. helm/git is highly recommended but not required.
Adding them to myks, would require some additional configuration options to overwrite versions, etc.

Comment (Zebradil) [reply docs/README.md:107]: True

**Pull request merged**